compiler_gym/service/connection.py

```diff
@@ -52,6 +52,8 @@
 class ConnectionOpts(BaseModel):
     """The options used to configure a connection to a service."""
 
+    # Deprecated.
+    # This is for backwards compatibility and will be removed in a future release.
     rpc_call_max_seconds: float = 300
     """The maximum number of seconds to wait for an RPC method call to succeed."""
```

Comment:
I'd suggest deleting this. We don't have any way to emit a deprecation warning to code, and now that this param is unused, it would silently break code that expected to change it. Best make that break explicit.

Reply:
Done!